meta-phosphor/common/recipes-phosphor/host-ipmid/host-ipmid.bb

```diff
@@ -17,7 +17,10 @@ TARGET_CFLAGS   += "-fpic"
 RDEPENDS_${PN} += "clear-once"
 RDEPENDS_${PN} += "settings"
 RDEPENDS_${PN} += "network"
-SRC_URI += "git://github.com/openbmc/phosphor-host-ipmid"
+SRC_URI += "git://github.com/openbmc/phosphor-host-ipmid \
+            file://host-ipmid-whitelist.conf \
+            file://ipmi-fru-parser-whitelist.conf \
```

Comment:
Did you figure out how to provide that config file by the ipmi-fru-parser package instead of this one?